tests/filecheck/parser-printer/builtin_attrs.mlir

```diff
@@ -174,7 +174,7 @@
                       value1 = dense<[true, false]> : tensor<2xi1>,
                       sym_name = "dense_bool_attr"} : () -> ()
 
-  // CHECK: "value1" = dense<[1, 0]> : tensor<2xi1>
+  // CHECK: "value1" = dense<[-1, 0]> : tensor<2xi1>
```

Comment:
this feels weird, I would expect us to print `true, false`, looking into this now

Reply:
This will likely be because these are being printed with `print_without_type`, which I expect has no `true`/`false` special casing

Reply:
or the data is being printed directly